README.md

```diff
@@ -52,6 +52,14 @@ module "gke" {
     }
   }
 
+  node_pools_metadata = {
+    all = {}
+
+    default-node-pool = {
+      shutdown-script = "${file(var.kube_shutdown_script)}"
```

Comment:
One small suggestion: let's just have a simple metadata annotation here instead of referencing a file + variable.

```suggestion
      node-pool-metadata-custom-value = "my-node-pool"
```